packages/cli/src/lib/cmds/init.ts

```diff
@@ -76,6 +76,13 @@ async function confirmTwaConfig(twaManifest: TwaManifest, prompt: Prompt): Promi
       validateColor,
   );
 
+  twaManifest.appVersionCode = await prompt.promptInput(
```

Comment:
I think it would make more sense for this question to come after the packageId question (they're both about Android specific details).

Reply:
Package ID on line 60?  moving to right after that.